src/components/AddNewBook.js

```diff
@@ -1,18 +1,56 @@
-import React from 'react';
+/* eslint-disable consistent-return */
```

Comment:
- [ ] Kindly remove this eslint disable rule. Its best to fix the issue in the code.

Reply:
Done